app/2fa/server/startup/settings.js

```diff
@@ -2,48 +2,65 @@ import { settings } from '../../../settings';
 
 settings.addGroup('Accounts', function() {
 	this.section('Two Factor Authentication', function() {
+		const enable2FA = {
+			_id: 'Accounts_TwoFactorAuthentication_Enabled',
+			value: true,
+		};
+
 		this.add('Accounts_TwoFactorAuthentication_Enabled', true, {
 			type: 'boolean',
 			public: true,
 		});
 		this.add('Accounts_TwoFactorAuthentication_MaxDelta', 1, {
```

Comment:
@sampaiodiego, will these changes require a migration? 👀 (the updates to the `enableQuery`)

Reply:
no.. `enableQuery` is always updated.

Reply:
Thanks!